be/src/exprs/vectorized/in_const_predicate.hpp

```diff
@@ -79,6 +80,18 @@ class VectorizedInConstPredicate final : public Predicate {
         return Status::OK();
     }
 
+    Status merge(Predicate* predicate) {
```

Comment:
The arg type should be `VectorizedInConstPredicate `

Reply:
dynamtic type is checked inside VectorizedInConstPredicate, VectorizedInConstPredicate is a class template.

Reply:
`merge()` should be annotate with `override`.